docs/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Windows image file caches
```

Comment:
Please move this file to the root of the solution

Reply:
done